trie/database.go

```diff
@@ -99,6 +99,15 @@ type rawNode []byte
 func (n rawNode) cache() (hashNode, bool)   { panic("this should never end up in a live trie") }
 func (n rawNode) fstring(ind string) string { panic("this should never end up in a live trie") }
 
+func (n rawNode) EncodeRLP(w io.Writer) error {
+	_, err := w.Write([]byte(n))
+	if err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
How about just 
```suggestion
	_, err := w.Write([]byte(n))
	return err
```

This change does modify a struct that's already used by the trie today, and apparently in a way which modifies the behaviour of `rawNode`. So, I'm curious why this works, but even more curious as to why this change won't break older code, supposedly relying on how it worked before... ?

Reply:
no "older" code was serializing `rawNode`, so no older code should be broken as far as I can tell